migrations/1719498613689-CascadeUserProfile.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CascadeUserProfile1719498613689 implements MigrationInterface {
```

Comment:
I think there is no sense to create such migration untill the PR is not merged 
Let's just move this logic to CreateProfileMigration
And on your local machine you can roll back last 3 migrations and apply then 2